common/models/user.js

```diff
@@ -319,7 +319,7 @@ module.exports = function(User) {
       options.port +
       options.restApiRoot +
       userModel.http.path +
-      userModel.confirm.http.path +
+      '/confirm' +
```

Comment:
This is changing configurable `confirm.http.path` to hard-coded `/confirm`, isn't that a breaking change?